### PR TITLE
Currency on Death

### PR DESCRIPTION
**Pull request by jakkeez**
When enemies die they now drop currency, the currency is specified by type and amount in currency component.

### CODE REVIEW

File: StortSpelProjekt/Game/src/main.cpp [hunk 1]
```diff
@@ -6,6 +6,9 @@
 #include "Player.h"
 #include "Shop.h"
 
+// Game components includes
+#include "Gamefiles/Components/CurrencyComponent.h"
```
Comment (kattalainen) [inline StortSpelProjekt/Game/src/main.cpp]: you can include this in GameComponents.h instead :>

File: StortSpelProjekt/Sandbox/src/main.cpp [hunk 2]
```diff
@@ -3,6 +3,7 @@
 #include "GameNetwork.h"
 #include "Player.h"
 #include "Shop.h"
+#include "Components/CurrencyComponent.h"
```
Comment (kattalainen) [inline StortSpelProjekt/Sandbox/src/main.cpp]: GameComponents.h !

**Review by kattalainen — DISMISSED** [review]
just move the currency includes to gamecomponent.h then it's fine

**Review by kattalainen — APPROVED** [review]

**Review by OtakuFreak13 — APPROVED** [review]
Runs and seems to add currency to player.

**Pull request merged**